vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/DashboardWidget.java

```diff
@@ -22,4 +22,23 @@
 @JsModule("@vaadin/dashboard/src/vaadin-dashboard-widget.js")
 // @NpmPackage(value = "@vaadin/dashboard", version = "24.6.0-alpha0")
 public class DashboardWidget extends Component {
+
+    /**
+     * Returns the title of the widget.
+     *
+     * @return the {@code widget-title} property from the web component
+     */
+    public String getTitle() {
+        return getElement().getAttribute("widget-title");
```

Comment:
Prefer using `setProperty` / `getProperty` over the attribute APIs

Reply:
Done.

Reply:
Since we're talking about the property in the API documentation, the name should be aligned with it: `...{@code widget-title} property` -> `{@code widgetTitle} property` (same change needed in the TB element)

Reply:
Right, fixed both.